binaryTreeMaximumPathSum.py

```diff
@@ -0,0 +1,25 @@
+# Definition for a binary tree node.
+# class TreeNode:
+#     def __init__(self, x):
+#         self.val = x
+#         self.left = None
+#         self.right = None
+
+class Solution:
+    def maxPathSum(self, root):
+        """
+        :type root: TreeNode
+        :rtype: int
+        """
+        return self.helper(root)[2]
+    def helper(self, root):
+        if not root:
+            return [0, 0, - 2**31 - 1]
+        leftVal = self.helper(root.left)
```

Comment:
When `helper` function returns an array, you can unpack it using
`[a, b, c] = self.helper(XXX);`
For values returned that you are not interested, you can use `_` and do something like
`[a, _, c] = self.helper(XXX)`